<?xml version="1.0" encoding="UTF-8"?>
<interface domain="easyeffects">
    <object class="GtkBox" id="top_box">
        <property name="valign">center</property>
        <property name="spacing">24</property>
        <property name="orientation">vertical</property>
        <child>
            <object class="GtkBox">
                <property name="valign">center</property>
                <property name="halign">center</property>
                <property name="spacing">48</property>
                <child>
                    <object class="GtkGrid">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="row-spacing">6</property>
                        <property name="column-spacing">6</property>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Show Spectrum</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">0</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkSwitch" id="show">
                                <property name="halign">start</property>
                                <property name="valign">center</property>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">0</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Fill</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">1</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkSwitch" id="fill">
                                <property name="halign">start</property>
                                <property name="valign">center</property>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">1</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Border</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">2</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkSwitch" id="show_bar_border">
                                <property name="halign">start</property>
                                <property name="valign">center</property>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">2</property>
                                </layout>
                            </object>
                        </child>
                    </object>
                </child>

                <child>
                    <object class="GtkGrid">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="row-spacing">6</property>
                        <property name="column-spacing">6</property>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Spectrum Type</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">0</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkComboBoxText" id="spectrum_type">
                                <property name="halign">start</property>
                                <items>
                                    <item translatable="yes">Bars</item>
                                    <item translatable="yes">Lines</item>
                                </items>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">0</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Spectrum Color</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">1</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkColorButton" id="spectrum_color_button">
                                <property name="halign">start</property>
                                <property name="use-alpha">True</property>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">1</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkLabel">
                                <property name="halign">end</property>
                                <property name="valign">center</property>
                                <property name="label" translatable="yes">Axis Color</property>
                                <layout>
                                    <property name="column">0</property>
                                    <property name="row">2</property>
                                </layout>
                            </object>
                        </child>
                        <child>
                            <object class="GtkColorButton" id="axis_color_button">
                                <property name="halign">start</property>
                                <property name="use-alpha">True</property>
                                <layout>
                                    <property name="column">1</property>
                                    <property name="row">2</property>
                                </layout>
                            </object>
                        </child>
                    </object>
                </child>
            </object>
        </child>

        <child>
            <object class="GtkGrid">
                <property name="halign">center</property>
                <property name="valign">center</property>
                <property name="row-spacing">6</property>
                <property name="column-spacing">24</property>
                <child>
                    <object class="GtkLabel">
                        <property name="halign">center</property>
                        <property name="valign">end</property>
                        <property name="label" translatable="yes">Points</property>
                        <layout>
                            <property name="column">0</property>
                            <property name="row">0</property>
                        </layout>
                    </object>
                </child>
                <child>
                    <object class="GtkSpinButton" id="n_points">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="orientation">vertical</property>
                        <property name="digits">1</property>
                        <property name="update-policy">if-valid</property>
                        <property name="adjustment">
                            <object class="GtkAdjustment">
                                <property name="lower">2</property>
                                <property name="upper">1000</property>
                                <property name="step-increment">1</property>
                                <property name="page-increment">10</property>
                            </object>
                        </property>
                        <layout>
                            <property name="column">0</property>
                            <property name="row">1</property>
                        </layout>
                    </object>
                </child>

                <child>
                    <object class="GtkLabel">
                        <property name="halign">center</property>
                        <property name="valign">center</property>
                        <property name="label" translatable="yes">Height</property>
                        <layout>
                            <property name="column">1</property>
                            <property name="row">0</property>
                        </layout>
                    </object>
                </child>
                <child>
                    <object class="GtkSpinButton" id="height">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="orientation">vertical</property>
                        <property name="digits">1</property>
                        <property name="update-policy">if-valid</property>
                        <property name="adjustment">
                            <object class="GtkAdjustment">
                                <property name="lower">1</property>
                                <property name="upper">1000</property>
                                <property name="value">100</property>
                                <property name="step-increment">1</property>
                                <property name="page-increment">10</property>
                            </object>
                        </property>
                        <layout>
                            <property name="column">1</property>
                            <property name="row">1</property>
                        </layout>
                    </object>
                </child>

                <child>
                    <object class="GtkLabel">
                        <property name="halign">center</property>
                        <property name="valign">center</property>
                        <property name="label" translatable="yes">Line Width</property>
                        <layout>
                            <property name="column">2</property>
                            <property name="row">0</property>
                        </layout>
                    </object>
                </child>
                <child>
                    <object class="GtkSpinButton" id="line_width">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="orientation">vertical</property>
                        <property name="update-policy">if-valid</property>
                        <property name="adjustment">
                            <object class="GtkAdjustment">
                                <property name="lower">0.1</property>
                                <property name="upper">100</property>
                                <property name="value">2</property>
                                <property name="step-increment">0.1</property>
                                <property name="page-increment">1</property>
                            </object>
                        </property>
                        <layout>
                            <property name="column">2</property>
                            <property name="row">1</property>
                        </layout>
                    </object>
                </child>

                <child>
                    <object class="GtkLabel">
                        <property name="halign">center</property>
                        <property name="valign">center</property>
                        <property name="label" translatable="yes">Minimum</property>
                        <layout>
                            <property name="column">3</property>
                            <property name="row">0</property>
                        </layout>
                    </object>
                </child>
                <child>
                    <object class="GtkSpinButton" id="minimum_frequency">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="orientation">vertical</property>
                        <property name="update-policy">if-valid</property>
                        <property name="adjustment">
                            <object class="GtkAdjustment">
                                <property name="lower">20</property>
                                <property name="upper">22000</property>
                                <property name="value">20</property>
                                <property name="step-increment">10</property>
                                <property name="page-increment">100</property>
                            </object>
                        </property>
                        <layout>
                            <property name="column">3</property>
                            <property name="row">1</property>
                        </layout>
                    </object>
                </child>

                <child>
                    <object class="GtkLabel">
                        <property name="halign">center</property>
                        <property name="valign">center</property>
                        <property name="label" translatable="yes">Maximum</property>
                        <layout>
                            <property name="column">4</property>
                            <property name="row">0</property>
                        </layout>
                    </object>
                </child>
                <child>
                    <object class="GtkSpinButton" id="maximum_frequency">
                        <property name="halign">center</property>
                        <property name="valign">start</property>
                        <property name="orientation">vertical</property>
                        <property name="width-chars">10</property>
                        <property name="update-policy">if-valid</property>
                        <property name="adjustment">
                            <object class="GtkAdjustment">
                                <property name="lower">20</property>
                                <property name="upper">22000</property>
                                <property name="value">20</property>
                                <property name="step-increment">10</property>
                                <property name="page-increment">100</property>
                            </object>
                        </property>
                        <layout>
                            <property name="column">4</property>
                            <property name="row">1</property>
                        </layout>
                    </object>
                </child>
            </object>
        </child>
    </object>

    <object class="GtkSizeGroup">
        <property name="mode">horizontal</property>
        <widgets>
            <widget name="n_points" />
            <widget name="height" />
            <widget name="line_width" />
            <widget name="minimum_frequency" />
            <widget name="maximum_frequency" />
        </widgets>
    </object>
</interface>